modules/rank-eval/src/test/java/org/opensearch/index/rankeval/DiscountedCumulativeGainTests.java

```diff
@@ -143,20 +143,20 @@ public void testDCGAtSixMissingRatings() {
         assertEquals(12.779642067948913, result.metricScore(), DELTA);
         assertEquals(2, filterUnratedDocuments(result.getHitsAndRatings()).size());
 
-        /**
-         * Check with normalization: to get the maximal possible dcg, sort documents by
-         * relevance in descending order
-         *
-         * rank | relevance | 2^(relevance) - 1 | log_2(rank + 1) | (2^(relevance) - 1) / log_2(rank + 1)
-         * ----------------------------------------------------------------------------------------
-         * 1 | 3 | 7.0 | 1.0  | 7.0
-         * 2 | 3 | 7.0 | 1.5849625007211563 | 4.416508275000202
-         * 3 | 2 | 3.0 | 2.0  | 1.5
-         * 4 | 1 | 1.0 | 2.321928094887362   | 0.43067655807339
-         * 5 | n.a | n.a | n.a.  | n.a.
-         * 6 | n.a | n.a | n.a  | n.a
-         *
-         * idcg = 13.347184833073591 (sum of last column)
+        /*
+          Check with normalization: to get the maximal possible dcg, sort documents by
+          relevance in descending order
+          <p>
```

Comment:
No need for `<p>` here (internal method comments)

Reply:
fixed in new commit

Reply:
Same, it is still there?

Reply:
no this one was fixed